src/main/java/xyz/nucleoid/plasmid/mixin/game/event/ScreenHandlerMixin.java

```diff
@@ -64,9 +64,10 @@ private void close(PlayerEntity player, CallbackInfo ci) {
 
     private boolean shouldBlockThrowingItems(PlayerEntity player, int slot, ItemStack stack) {
         ManagedGameSpace gameSpace = ManagedGameSpace.forWorld(player.world);
-        if (gameSpace != null && gameSpace.containsPlayer((ServerPlayerEntity) player)) {
+	    ServerPlayerEntity serverPlayer = (ServerPlayerEntity) player;
```

Comment:
Hm, indentation seems odd here still

Reply:
Fixed in defeb971c1d89c7330a7c0665a36bc1c8e405774